src/modules/reporting/actions/estimate-submit-market-contribute.js

```diff
@@ -0,0 +1,28 @@
+import { augur } from 'services/augurjs'
+import logError from 'utils/log-error'
+import noop from 'utils/noop'
+
+export const estimateSubmitMarketContribute = (marketId, amount, callback = logError) => (dispatch, getState) => {
+  const { loginAccount, marketsData } = getState()
+
+  const market = marketsData[marketId]
+  if (!market) return callback('Market not found')
+  const payoutNumerators = Array(market.numOutcomes).fill(0)
+  payoutNumerators[0] = market.numTicks
+
+  augur.api.Market.contribute({
+    meta: loginAccount.meta,
+    tx: { to: marketId, estimateGas: true, gas: '0x632ea0' },
```

Comment:
I asked @pgebheim to add a constant called DEFAULT_MAX_GAS for this a few weeks ago. Please use if appropriate.